kafka/consumer/group.py

```diff
@@ -63,6 +63,13 @@ class KafkaConsumer(six.Iterator):
             the server will block before answering the fetch request if
             there isn't sufficient data to immediately satisfy the
             requirement given by fetch_min_bytes. Default: 500.
+        fetch_max_bytes (int): The maximum amount of data the server should
+            return for a fetch request. This is not an absolute maximum, if the
+            first message in the first non-empty partition of the fetch is
+            larger than this value, the message will still be returned to
+            ensure that the consumer can make progress. NOTE: consumer performs
+            multiple fetches in parallel so memory usage will be higher.
```

Comment:
This note is a great idea, but tweak the wording slightly? It's unclear 1) what it will be higher than , and 2) how much higher... are we talking 20%, 2x, 5x, etc?